.eslintrc.js

```diff
@@ -0,0 +1,34 @@
+module.exports = {
+  root: true,
+  env: {
+    browser: true,
+    node: true,
+    test: true,
+    cypress: true
+  },
+  parserOptions: {
+    parser: '@typescript-eslint/parser'
+  },
+  extends: [
+    '@nuxtjs',
+    'prettier',
+    'prettier/vue',
+    'plugin:prettier/recommended',
+    'plugin:nuxt/recommended'
```

Comment:
Maybe we should add it. Do you want to do that?

Reply:
Looks like a good thing; https://eslint.vuejs.org/user-guide/#installation

Strange it's not included(!?) or maybe it's similar to nuxt/recommended; https://github.com/nuxt/eslint-plugin-nuxt

Reply:
I've taken a closer look now, and both seems to be usable.

But if I try to use vue/recommended it wants to "vue/html-self-closing" and change `<img />` to `<img>` and if I change that, prettier will complain instead.

I also think vue/singleline-html-element-content-newline will cause problems because it introduce whitespaces into divs, it might be a bug, but I had some problems with it.

Do you like to try? 

I guess prettier (without vue) should be removed.

I'm using these from vue-cli:
````
      "plugin:vue/recommended",
      "@vue/prettier",
      "@vue/typescript"
````

Reply:
Added it 👍